ArcGISRuntimeSDKQt_QMLSamples/Analysis/FindServiceAreasForMultipleFacilities/FindServiceAreasForMultipleFacilities.qml

```diff
@@ -0,0 +1,147 @@
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.8
+import QtQuick.Layouts 1.11
+import QtQuick.Controls 2.2
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property url featureTableUrl: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0"
+    property url imageUrl: "https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png"
+    property url serviceTaskAreaUrl: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ServiceArea"
+    property list<SimpleFillSymbol> fillSymbols: [
+        SimpleFillSymbol {
+            style: Enums.SimpleFillSymbolStyleSolid
+            color: "#66FFA500"
+            outline: null
+        },
+        SimpleFillSymbol {
+            style: Enums.SimpleFillSymbolStyleSolid
+            color: "#66FF0000"
+        }
+    ]
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        GraphicsOverlay {
+            id: serviceAreasOverlay
+        }
+
+        Map {
+            BasemapLightGrayCanvas {}
+
+            // create query parameters used to select all facilities from the feature table
+            QueryParameters {
+                id: queryParameters
+                whereClause: "1=1"
+            }
+
+            FeatureLayer {
+                featureTable: ServiceFeatureTable {
+                    id: featureTable
+                    url: featureTableUrl
+                }
+                onComponentCompleted: {
+                    load(); }
+                onLoadStatusChanged: {
+                    if(loadStatus === Enums.LoadStatusLoaded) {
+                        mapView.setViewpointGeometryAndPadding(fullExtent, 100);
+                    }
+                }
+                onErrorChanged: {
+                    console.warn(error.message); }
+
+                renderer: SimpleRenderer {
+                    PictureMarkerSymbol {
+                        url: imageUrl
+                        height: 25
+                        width: 25
+                    }
+                }
+            }
+
+            ServiceAreaTask {
+                id: serviceAreaTask
+                url: serviceTaskAreaUrl
+
+                onErrorChanged: {
+                    console.warn(error.message); }
```

Comment:
bracket on new line. I won't comment on any more of these

Reply:
Was this fixed?